src/S3Iterator.cpp

```diff
@@ -1,165 +1,7 @@
 #include "S3Iterator.h"
```

Comment:
Why is this file still here? It doesn't seem to provide any meaningful purpose. Same with S3Iterator.h